packages/core/src/lib/markdown-it/plugins/markdown-it-alt-frontmatter.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+// Process "---" => "<frontmatter>
+
+module.exports = function alt_frontmatter_plugin(md) {
+  function alt_frontmatter(state, startLine, endLine, silent) {
```

Comment:
I notice that `silent` is never used - is it possible to remove it (or prefix it with `_` if it can't be removed)?